portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,56 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random Bible verse to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let bibleVerses = new Map();
+bibleVerses.set('Romans 5: 3-5', 'Not only so, but we also glory in our ' +
+		'sufferings, because we know that suffering produces perseverance; ' +
+		'perseverance, character; and character, hope. And hope does not put us ' +
+		'to shame, because God’s love has been poured out into our hearts ' +
+		'through the Holy Spirit, who has been given to us.');
+bibleVerses.set('Romans 10:13', 'For, “Everyone who calls on the name of ' +
+		'the Lord will be saved.”');
+bibleVerses.set('Psalms 94:19', 'When anxiety was great within me, your ' +
+		'consolation brought me joy.');
+bibleVerses.set('John 3:16', 'For God so loved the world that he gave his ' +
+		'one and only Son, that whoever believes in him shall not perish but ' +
+		'have eternal life.');
+bibleVerses.set('James 1:5', 'If any of you lacks wisdom, you should ask ' +
+		'God, who gives generously to all without finding fault, and it will be ' +
+		'given to you.');
+bibleVerses.set('Joshua 1:9', 'Have I not commanded you? Be strong and ' +
+		'courageous. Do not be afraid; do not be discouraged, for the Lord your ' +
+		'God will be with you wherever you go.');
+
+function getRandomBibleVerse() {
```

Comment:
Would be worth getting in the habit of documenting your function with function-level comments - https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
Cool, already added it. Don't know if I did it all correctly though (for example, I omitted @param because my functions had no parameters).

Reply:
Yeah looks good to me!